src/auth/index.js

```diff
@@ -543,4 +543,39 @@ utils.wrapPropertyMethod(
  */
 utils.wrapPropertyMethod(AuthenticationClient, 'passwordGrant', 'oauth.passwordGrant');
 
+/**
+ * Sign in using a refresh token
+ *
+ * @method    refresh
+ * @memberOf  module:auth.OAuthAuthenticator.prototype
```

Comment:
I think this was copy/pasted from somewhere else, right?

Reply:
Yep, hastily.